serving/src/main/java/feast/serving/config/ServingServiceConfig.java

```diff
@@ -82,6 +87,32 @@ public ServingService servingService(
     return servingService;
   }
 
+  @Bean
+  public HistoricalRetriever createJdbcHistoricalRetriever(FeastProperties feastProperties) {
+    FeastProperties.Store store = feastProperties.getActiveStore();
+    Map<String, String> config = store.getConfig();
```

Comment:
Shall we somehow convert the `config` into `JdbcConfig`? I can see there is a `getUsername()` method under `JdbcConfig`.

Reply:
Not sure what you mean by that, `JdbcConfig` in what package?

Reply:
Currently, JdbcFeatureSink has `JdbcConfig` as a field, at [here](feast/storage/connectors/jdbc/writer/JdbcFeatureSink.java).
Where the `JdbcConfig` is under [feast.core.Stor.proto. ](feast/core/Store.proto).
```
  message JdbcConfig {
    string url = 1;
    string class_name = 2;
    int32 batch_size = 3;
    string username = 4;
    string password = 5;
    string staging_location = 6;
    string database = 7;
    string schema = 8;
    string warehouse = 9;
  }
```
So, I'm wondering if it is possible to use JdbcConfig on the retriever side as well?
It seems like `Map<String, String> config = store.getConfig();` will get the store's config as a map in this case? And we can just use this config map and config.get("class_name") to get the `className`?

Reply:
Ideally it shouldn't be a proto, since this information is not passed over the network. Not sure why they made it proto. I will check that code.